docs/samples/python/presidio_notebook.ipynb

```diff
@@ -152,9 +156,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "converted_analyzer_results = [result.to_dict() for result in analyzer_results]\n",
+    "new_results = [AnalyzerResult(result.entity_type, result.start, result.end, result.score) for result in analyzer_results]\n",
```

Comment:
I think we can skip this part and just pass the analyzer results directly to the AnonymizerEngine

Reply:
analyzer and anonymizer dont have common classes, so we need to convert the analyzer results (list of RecognizerResult) to a anonymizer expected input (list of AnalyzerResult)

Reply:
It would work even without it. wild wild west Python 😊